.pre-commit-config.yaml

```
@@ -136,7 +136,7 @@ repos:
     hooks:
       - id: flake8
         additional_dependencies:
-          - flake8-docstrings
+          - flake8-docstrings>=1.7.0
```

Since this package is not updated frequently, and hasn't been updated in over a year, I think it's safe to pin to a specific version. This would also align with [what has been done with `prettier`](https://github.com/cisagov/skeleton-generic/blob/7ddea47402e48fbbcc470bdba96a627783f77e68/.pre-commit-config.yaml#L45-L49) and how hooks are pinned with pre-commit in general.

```suggestion
          - flake8-docstrings==1.7.0
```